notebooks/spark_basics/DataFrame.ipynb

```diff
@@ -11,12 +11,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Spark SQL provides a special type of RDD called *DataFrame* which is organized into named columns. It is conceptually equivalent to a table in a relational database or a data frame in R/Python, but with richer optimizations. DataFrames can be constructed from a wide array of sources such as: structured data files, tables in Hive, external databases, or existing RDDs. In the following example, we will show that how to create a Dateframe for a JSON dataset."
+    "Spark SQL provides a special type of RDD called [*DataFrame*](https://spark.apache.org/docs/latest/sql-programming-guide.html#datasets-and-dataframes) which is organized into named columns. It is conceptually equivalent to a table in a relational database or a data frame in R/Python, but with richer optimizations. DataFrames can be constructed from a wide array of sources such as: structured data files, tables in Hive, external databases, or existing RDDs. In the following example, we will show that how to create a Dateframe for a JSON dataset."
```

Comment:
DataFrame is not RDD; it's A is a distributed collection of data organized into named columns, built on top of RDD

Reply:
fixed under commit [#13d2ed2 change the dataframe definition](https://github.com/intel-analytics/BigDL-Tutorials/pull/37/commits/13d2ed203d3ebac96579da944e0a4a5c9346f406)